scripts/test-util.js

```diff
@@ -0,0 +1,297 @@
+'use strict';
```

Comment:
nit: I'd suggest to rename this script to `test-runner.js` as "test-util" makes an impression that this script contains various utility functions used in tests.